src/utils/connectDB.ts

```diff
@@ -0,0 +1,10 @@
+import { MongoClient } from 'mongodb';
+
+export async function connectToDatabase() {
+	const client = await MongoClient.connect((process.env.MONGODB_URI), {
+		useNewUrlParser: true,
+		useUnifiedTopology: true,
+	});
+
+	return client;
+}
```

Comment:
there is already a mongodb connection function in `/utils/store`. I would recommend using mongoose though. Take a look at `src/pages/api/user/[id].ts` for reference